ansible/group_vars/all.yml

```diff
@@ -11,3 +11,8 @@ bootstrap__consul_certs_client_key: '{{lookup("bitwarden", "consul/certs",
 bootstrap__sshguard_whitelist_extra: ['{{lookup("bitwarden", "sshguard/whitelist", field="jakubgs-home")}}']
 # Wireguard
 wireguard_consul_acl_token: '{{lookup("bitwarden", "consul/acl-tokens", field="wireguard")}}'
+
+# Custom SSH accounts for Nimbus fleet, should start from UID 8000.
+bootstrap__active_extra_users:
+  - { name: hanno,  uid: 8003, admin: true, key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDDlvmNGKxoddAmOuXvjm0II0M82aLjczb8F1ZTSV7zyvKaw6DNuaN6c6ZCdUvdjfF4hdEwMXNB37Cr5oysPo41rIuii+RVPd+c2WLZnC/MCg8d6b0/mREIfpuEMlz+u4lyr8/DST7zO4Ke95w5kVGtlh2kR88F9mlJlkyq2NCiqBU/blKObOjqS8OMRuMJ0GpwzF9+/dfXVahzdgHgKS0Q1ATvLHL0DvZSAGwHfHcIV5nF0ddRCofX4L2shIImZk5dAsATQTkT2gNWObhF6KuWbVyuhsLfRletzdnf8jUYm9Uatf7Woa9CxbPjdGxVxniZblmOumcaCNgaioKs0qeZzwNQmgJ/PXXw9uVWhDUYvNh7Cz+SNTVm4WG1tEk3WO9EU5dcCTgfEa94LUl5G+yXYX8H00spubpiWXv/0RjEQX4CZeu0pff209GLDWdpIaV8p9QVFBr8X8t8jw5zw5j/PxkprcT5P2A1t+WOUn9LrBzOJ1iAU3oJV3ZMYQhUS78= hanno@status.im' }
+  - { name: ivan,   uid: 8004, admin: true, key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC5FLTJPOAhHRaWbZCUXH05vbMDh/rF+FoTrZK7jRYK0TZwDeYqrTbkpVB9xzRFI0llh3NGWS5ZXde5mcpoDIkEeHSBKMZ9UHppqOmd1ha3XRMonh+XtwoO8Hky9ouemJ8hNlcOkkDGsBVj/Rj3ebuEXNY8Bzqju+SQn9RpFTkPbwnfRvwbX/nCMFbftOs8M13L9dwgPO4CFUERvrl7DGxcIxv+xB+yqsv8KaFb9wwutPqwSoO0GCqZwQd22am7n4KrFbd1+WJijg6LT4/iAkyBnqY9ciniG4V0hgGce3/jUIw9T0wx3VS7kbPJNqRMC0dBM0UgIwGKQmLPvZbGpFYB cardno:000623466796' }
```

Comment:
Please use your email, not `cardno:000623466796`.

Reply:
Sure, I will. I used the same entry that I used for `status` fleets. Shall I change it there as well?